src/include/cluster/dialogs.rb

```diff
@@ -198,16 +199,25 @@ def ValidateCommunication
       if UI.QueryWidget(Id(:transport), :Value) == "udpu"
         i = 0
         Builtins.foreach(Cluster.memberaddr) do |value|
-          if  !IP.Check(value[:addr1]) || ( UI.QueryWidget(Id(:enable2), :Value) && !IP.Check(value[:addr2]) )
-            UI.ChangeWidget(:memberaddr, :CurrentItem, i)
-            i = 0
-            raise Break
+          if ip == "ipv4"
```

Comment:
Could use a wrapper function `IPCheck` to hide the IPv4/v6 details. The function can be reused multiple times.